src/query/service/src/sql/planner/binder/insert.rs

```diff
@@ -12,54 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::borrow::Cow;
-use std::io::Cursor;
-use std::ops::Not;
 use std::sync::Arc;
 
-use common_ast::ast::Expr;
 use common_ast::ast::InsertSource;
 use common_ast::ast::InsertStmt;
 use common_ast::ast::Statement;
-use common_ast::parser::parse_comma_separated_exprs;
-use common_ast::parser::tokenize_sql;
-use common_ast::Backtrace;
-use common_datablocks::DataBlock;
-use common_datavalues::prelude::*;
-use common_datavalues::DataSchemaRef;
 use common_datavalues::DataSchemaRefExt;
-use common_datavalues::DataType;
-use common_datavalues::TypeDeserializer;
-use common_datavalues::TypeDeserializerImpl;
-use common_exception::ErrorCode;
 use common_exception::Result;
-use common_formats::FormatFactory;
-use common_io::prelude::BufferReader;
-use common_io::prelude::*;
-use common_pipeline_transforms::processors::transforms::Transform;
 use tracing::debug;
 
 use crate::clusters::ClusterHelper;
```

Comment:
Do we need `ClusterHelper`?

Reply:
enable_distributed_optimization: !self.ctx.get_cluster().is_empty(),
   |                                                                              ^^^^^^^^ method not found in `std::sync::Arc<Cluster>`